cookbooks/aws-parallelcluster-platform/resources/intel_hpc/intel_hpc.rb

```diff
@@ -22,5 +22,71 @@
 end
 
 action :configure do
-  # do nothing
+  return unless intel_hpc_supported? && (node['cluster']['install_intel_base_toolkit'] == "true" || node['cluster']['install_intel_hpc_toolkit'] == "true" || node['cluster']['install_intel_python'] == "true")
+
+  base_toolkit_version = "2023.2.0.49397"
```

Comment:
Why not defining them as cookbook attributes?

Reply:
With this commit https://github.com/aws/aws-parallelcluster-cookbook/commit/b837b5f242158448c5b6d6902066430da4fd3947, I feel like we are moving away from a centralized attribute file?

Reply:
That's interesting, thanks for sharing. I'll check the reason behind this approach. Regarding this specific Pr, let's proceed.